### PR TITLE
Add 2021 cfp extension blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @chkumar246 @vipulgupta2048

### CODE REVIEW

File: content/blog/2021/cfp-extension.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Title: PyCon India Call for Proposals is being extended!
+Date: 2020-05-05 17:00
+Category: 2021
+Slug: cfp-extension
```
Comment (anistark) [inline content/blog/2021/cfp-extension.md:4]: Slug let's make it `cfp-2021-extension` instead?

File: content/blog/2021/cfp-extension.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Summary: PyCon India 2021 call for proposal is being extended till May 22 due to the circumstances.
+
+We hope this announcement finds you and your family health and safe.
```
Comment (anistark) [inline content/blog/2021/cfp-extension.md:7]: health -> healthy

File: content/blog/2021/cfp-extension.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+PyCon India 2021 call for proposals (CFP) has received amazing talks and workshops from the global Python community. The deadline for the CFP was May 7th, 2021.
```
Comment (anistark) [inline content/blog/2021/cfp-extension.md:9]: The language seems odd. Please revisit.

**Review by anistark — CHANGES_REQUESTED** [review]

File: content/blog/2021/cfp-extension.md (hunk 1 above)
Comment (Sc0rpi0n101) [reply content/blog/2021/cfp-extension.md:4]: It will be `blog/2021/cfp-extension` anyway

File: content/blog/2021/cfp-extension.md (hunk 3 above)
Comment (Sc0rpi0n101) [reply content/blog/2021/cfp-extension.md:9]: Sure

Comment (Sc0rpi0n101) [reply content/blog/2021/cfp-extension.md]: @anistark Check now.

**Review by raukadah — APPROVED** [review]

**Review by anistark — APPROVED** [review]

**Pull request merged**